script/publish.js

```diff
@@ -0,0 +1,36 @@
+const path = require("path");
```

Comment:
Can we do something like `#! /usr/bin/env node`, then we can call it with `       - run: ./script/publish` so it's similar to `script/release`